lib/cluster.js

```diff
@@ -636,6 +638,8 @@ function workerInit() {
       if (key === undefined) return;
       send({ act: 'close', key: key });
       delete handles[key];
+      const indexesKey = key.substring(0, key.lastIndexOf(':'));
+      indexes[indexesKey] = undefined;
```

Comment:
> This looks too hacky. I'd rename the const key = ... at the top of cluster._getServer() to const indexesKey and use that here.

Maybe I'm missing something, but how would I do that? They're in different contexts. I can pass it to the `shared` and `rr` functions as an argument though.

> Also, this would be a place where the delete operator is warranted, otherwise indexes can grow unchecked.

You're right. Will fix.

Thanks!